deploy/helm/sumologic/values.yaml

```
@@ -772,6 +772,12 @@ fluent-bit:
   ## Setting the flag to true instead of commenting out will break other functionality.
   # enabled: false
 
+  ## Configuration for fluent-bit service
+  service:
+    ## Add custom labels to fluent-bit service
+    labels:
+      sumologic.com/scrape: "true" #  label used by fluent-bit service monitor
```

Shouldn't be this way? 🤔 

```suggestion
      sumologic.com/scrape: "true"  # label used by fluent-bit service monitor
```